src/confluent_kafka/admin/__init__.py

```diff
@@ -787,3 +787,25 @@ def alter_consumer_group_offsets(self, alter_consumer_group_offsets_request, **k
         super(AdminClient, self).alter_consumer_group_offsets(alter_consumer_group_offsets_request, f, **kwargs)
 
         return futmap
+
+    def set_sasl_credentials(self, username, password):
+        """
+        Sets the SASL credentials used for this client.
+        These credentials will overwrite the old ones, and will be used the
+        next time the client needs to authenticate.
+        This method will not disconnect existing broker connections that
+        have been established with the old credentials.
+        This method is applicable only to SASL PLAIN and SCRAM mechanisms.
+
+        :param str username: The username to set.
+        :param str password: The password to set.
+
+        :returns: A dict of futures for each group, keyed by the group id.
```

Comment:
I kind of see there is a disparity between returns and rtype. Didn't quite get why it returns ConsumerGroupTopicPartitions.

Reply:
Left it there accidentally, it just returns None